### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'event'

### DISCUSSION THREAD

**Discussion by plearatblast** [ourownstory/neural_prophet#553] [Q&A - get help using NeuralProphet] [answered]
Hello,
I'm trying to add a "holiday" event to handle a "covid influenced" period. I get the _AttributeError: 'NoneType' object has no attribute 'event'_  error when making future dataframe. It's probably something I overlooked but assistance is welcome.

```
df = pd.read_csv('data.csv')
events_df = pd.DataFrame({
    'event': 'covid',
    'ds':  pd.date_range(start='2020-03-15', end='2020-05-01', freq='D')
    })
m = NeuralProphet(loss_func="MSE")
m = m.add_events(["covid"])
history_df = m.create_df_with_events(df, events_df)
metrics = m.fit(history_df, freq="D")
future = m.make_future_dataframe(history_df, periods=30)
forecast = m.predict(future)
```

**Comment by plearatblast** [comment, marked as answer]
Solved this with the explicit syntax:
`future = m.make_future_dataframe(df=history_df, events_df=events_df, periods=30, n_historic_predictions=len(df))
forecast = m.predict(df=future)`

**Reply by ourownstory** [reply]
Great! you were just missing `events_df=events_df` in `make_future_dataframe`
